lacework/resource_lacework_integration_gcp_agentless_scanning.go

```diff
@@ -0,0 +1,384 @@
+			},
+			"credentials": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"client_id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"private_key_id": {
+							Type:     schema.TypeString,
+							Required: true,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+								return !d.HasChanges(
+									"name", "resource_level", "resource_id", "org_level", "enabled",
+									"credentials.0.client_id",
+									"credentials.0.client_email",
+								)
+							},
+						},
+						"client_email": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"private_key": {
+							Type:      schema.TypeString,
+							Required:  true,
+							Sensitive: true,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+								// @afiune we can't compare this element since our API, for security reasons,
+								// does NOT return the private key configured in the Lacework server. So if
+								// any other element changed from the credentials then we trigger a diff
+								return !d.HasChanges(
+									"name", "resource_level", "resource_id", "org_level", "enabled",
+									"credentials.0.client_id",
+									"credentials.0.client_email",
+								)
+							},
+						},
+						"token_uri": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "https://oauth2.googleapis.com/token",
+						},
+					},
+				},
+			},
+			"resource_level": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  api.GcpProjectIntegration.String(),
+				StateFunc: func(val interface{}) string {
+					return strings.ToUpper(val.(string))
+				},
+				ValidateFunc: func(value interface{}, key string) ([]string, []error) {
+					switch strings.ToUpper(value.(string)) {
+					case api.GcpProjectIntegration.String(),
+						api.GcpOrganizationIntegration.String():
+						return nil, nil
+					default:
+						return nil, []error{
+							fmt.Errorf("%s: can only be either '%s' or '%s'",
+								key,
+								api.GcpProjectIntegration.String(),
+								api.GcpOrganizationIntegration.String()),
+						}
+					}
+				},
+			},
+			"resource_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"created_or_updated_time": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_or_updated_by": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"type_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"org_level": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"bucket_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"scanning_project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"scan_frequency": {
```

Comment:
Can you add `Description`s for each of these? You can use the same descriptions for the AWS integration: https://github.com/lacework/terraform-provider-lacework/blob/07b1fa8fc26cce60b167fd521329514c85320510/lacework/resource_lacework_integration_aws_org_agentless_scanning.go#L41

Reply:
Thanks. Added short description for GCP fields too.